pandas/io/json/_normalize.py

```diff
@@ -228,14 +229,23 @@ def json_normalize(
     Returns normalized data with columns prefixed with the given string.
     """
 
-    def _pull_field(js, spec):
-        result = js
+    def _pull_field(js: Dict[str, Any], spec: Union[List, str]) -> Iterable:
+        result = js  # type: ignore
         if isinstance(spec, list):
             for field in spec:
                 result = result[field]
         else:
             result = result[spec]
 
+        if not isinstance(result, Iterable):
+            if isnull(result):
+                result = []  # type: ignore
+            elif is_scalar(result):
```

Comment:
On second read might be able to get rid of the `elif` and just keep to an `else`